js/src/Ice/ConnectionI.js

```diff
@@ -122,7 +121,7 @@ export class ConnectionI {
         this._initialized = false;
         this._validated = false;
 
-        this._asyncRequests = new HashMap(); // Map<int, OutgoingAsync>
```

Comment:
We only need to use HashMap when we need to use a custom key comparer. Here we can use the standard JavaScript Map.